src/shilpa/PrintNosWhileLoop.java

```diff
@@ -0,0 +1,69 @@
+//Print Number range using while loop.
+package shilpa;
+
+public class PrintNosWhileLoop{
+	void evenNumbers(int num1, int num2) {
+		System.out.println("Even Numbers from given range "+num1 + "-"+num2);
+		int index=num1;
```

Comment:
you don’t require to create extra variable, you can handle this without **index**, think on this

please refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files